README.md

```diff
@@ -303,13 +303,14 @@ Hurray! You successfully made a contribution!
   - [Jessica Salbert](https://github.com/jessicasalbert)
   - [Jivthesh](https://github.com/jivthesh)
   - [Johanna](https://github.com/Johanna-hub)
+  - [John Daniel](https://github.com/johndanny98)
   - [Jome Favourite](https://github.com/jomefavourite)
   - [Jonathan Niklasson Godar](https://github.com/JonathanGodar)
   - [Jose Angel Munoz](https://github.com/imjoseangel)
   - [Joseph Jose](https://github.com/josephjosedev)
   - [Josh Callahan](https://github.com/joshcallahan)
   - [Joykishan Sharma](https://github.com/JoykishanSharma)
-
+  
```

Comment:
```suggestion

```

Reply:
@Panquesito7  removed the space and  done commit to the branch

Reply:
@Panquesito7  thanks  a lot for helping me

Reply:
i had already  commit  my  github name through  another pull request  and @Panquesito7  should not merge this pull request